scripts/kafka-integration-test.sh

```diff
@@ -18,7 +18,7 @@ start_kafka() {
     -e KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT \
     -e KAFKA_CFG_CONTROLLER_LISTENER_NAMES=CONTROLLER \
     -e KAFKA_CFG_INTER_BROKER_LISTENER_NAME=PLAINTEXT \
-    bitnami/kafka:3.6
+    bitnami/kafka:3.6.0
```

Comment:
unrelated change, but CI was failing for this PR, because  bitnami/kafka:3.6.1 was released today and it's failing to start with 
```
Exception in thread "main" org.apache.kafka.common.config.ConfigException: Invalid value  for configuration message.max.bytes: Not a number of type INT
	at org.apache.kafka.common.config.ConfigDef.parseType(ConfigDef.java:745)
	at org.apache.kafka.common.config.ConfigDef.parseValue(ConfigDef.java:493)
	at org.apache.kafka.common.config.ConfigDef.parse(ConfigDef.java:486)
	at org.apache.kafka.common.config.AbstractConfig.<init>(AbstractConfig.java:112)
	at org.apache.kafka.common.config.AbstractConfig.<init>(AbstractConfig.java:145)
	at kafka.server.KafkaConfig.<init>(KafkaConfig.scala:1601)
	at kafka.server.KafkaConfig.<init>(KafkaConfig.scala:1603)
	at kafka.tools.StorageTool$.$anonfun$main$1(StorageTool.scala:50)
	at scala.Option.flatMap(Option.scala:271)
	at kafka.tools.StorageTool$.main(StorageTool.scala:50)
	at kafka.tools.StorageTool.main(StorageTool.scala)
```